ngrinder-controller/src/main/java/org/ngrinder/script/service/FileEntryService.java

```diff
@@ -85,24 +88,23 @@
 
 	private Cache fileEntryCache;
 
+	@Value("classpath:gitconfig_template/gitconfig.yml")
+	private Resource gitConfigTemplate;
```

Comment:
Please do not copy it from classpath. instead copy it from somewhere in .ngrinder folder